certified-connectors/AssemblyAI/apiDefinition.swagger.json

```diff
@@ -4185,16 +2978,28 @@
           "x-ms-summary": "Context"
         },
         "final_model": {
-          "description": "The model that is used for the final prompt after compression is performed.\nDefaults to \"default\".",
+          "description": "The model that is used for the final prompt after compression is performed.",
           "x-ms-visibility": "important",
           "type": "string",
+          "enum": [
```

Comment:
Kindly confirm that the existing flows will not be broken from the below swagger validation error.

The list of enum values has been added to the input field. Existing flows may use values outside of this list. ValuePath: operation/LemurTask/parameter/body/schema/final_model/enum